runtime/interpreter/value.go

```diff
@@ -15823,6 +15829,9 @@ func (v *DictionaryValue) Transfer(
 			panic(ExternalError{err})
 		}
 
+		elementMemoryUse := common.NewAtreeMapPreAllocatedElementsMemoryUsage(v.dictionary.Count())
```

Comment:
Same reason as previous comment.  

I think `elementSize` is computed and cached in `newDictionaryValueFromConstructor()`.

```suggestion
		elementMemoryUse := common.NewAtreeMapPreAllocatedElementsMemoryUsage(v.dictionary.Count(), v.elementSize)
```

Reply:
ah, I didn't know the element size was variable. Thanks for the suggestion! 👌